src/mame/mame.lst

```diff
@@ -31818,6 +31818,9 @@ komocomo                        //
 maddonnab                       // (c) 1995 Tuning
 oneshot                         // no copyright notice
 
+@source:misc/orientp.cpp
+east8                           //
+east8a                          //
 @source:misc/onetwo.cpp
```

Comment:
This file needs to be kept sorted, empty comments are useless, and formatting/spacing should be followed.